File: test/java/com/rustyrazorblade/cdm/InstallCommandTest.java
package com.rustyrazorblade.cdm;

import org.junit.Test;

/**
 * Created by jhaddad on 10/7/16.
 */
public class InstallCommandTest {
    @Test
    public void testLocalInstall() {
        String[] args = new String[]{"install", "."};
        CommandParser p = new CommandParser(args);
    }
}

File: test/kotlin/com/rustyrazorblade/cdm/InstallCommandTest.kt
package com.rustyrazorblade.cdm

import org.junit.Test

/**
 * Created by jhaddad on 10/7/16.
 */
class InstallCommandTest {
    @Test
    fun testLocalInstall() {
        val args = arrayOf("install", ".")
        val p = CommandParser(args)
    }
}
